nixos/modules/services/networking/tinc.nix

```diff
@@ -375,7 +385,9 @@ in
         path = [ data.package ];
         restartTriggers = [ config.environment.etc."tinc/${network}/tinc.conf".source ];
         serviceConfig = {
-          Type = "simple";
+          Type = if data.notify then "notify" else "simple";
+          NotifyAccess = "all";
+          PIDFile = "/run/tinc.${network}.pid";
```

Comment:
Shouldn't be this under /run/tinc/... ? Also this seems unrelated to me.

Reply:
No opinion on where it _should_ be, I am just configuring it to be the same as the existing `--pidfile /run/tinc.${network}.pid` in the `ExecStart` line further down.

But you're right, according to current https://www.freedesktop.org/software/systemd/man/systemd.service.html it shouldn't be needed:

For `Type=forking` only:

> recommended to also use the PIDFile= option, so that systemd can reliably identify the main process of the service

and

> Note that PID files should be avoided in modern projects. Use `Type=notify` or `Type=simple` where possible, which does not require use of PID files to determine the main process of a service and avoids needless forking.

So it seems to be unnecssary.

Reply:
Fixed.